colMike/Questioner_App
# #163069759 Create User register Api endpoint and its test

## What does this PR do ?
This pull request creates an endpoint that allows a user to register his/her details

Description of tasks to be completed?
Create an endpoint that allows the user to register an account under his/her details

Use Postman to test for full functionality

How should you manually test this?
Step 1

Navigate to your working directory, create and activate virtual environment

$ virtualenv venv

$ source venv/bin/activate

Clone the repository

$ git clone https://github.com/colMike/Questioner_App.git

Install project dependencies

pip install -r requirements.txt

Step 2

Running the application
$ flask run

Testing
$ pytest app/api/tests/v1

 Postman can also be used to test the route.

Prerequisites
Pytest, Postman, Git, Virtualenv

What are the relevant PT stories?
A user should be able to create an account #163069759

Screenshots
![user sign in](https://user-images.githubusercontent.com/19667042/50954920-9af0b300-146c-11e9-95ef-4636e3261c5a.PNG)
![unittest sign in](https://user-images.githubusercontent.com/19667042/50958364-61707580-1475-11e9-8b50-a11caf4e8c1d.PNG)
